pkg/backup/backuper_test.go

```diff
@@ -0,0 +1,184 @@
+package backup
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBackuperDefault(t *testing.T) {
+	// substitutes the open function to a mocked version
+	open = func(uri string) (DB, error) {
```

Comment:
This mocked implementation of `open` creates a `tempDatabase`. The `tempDatabase` is just regular `db` but it implements a special `Close` method that removes the database file. This is good for testing. It means we can create databases and destroy using the `DB` API.